src/lib.rs

```diff
@@ -87,6 +87,7 @@ pub mod dht11 {
     use super::*;
 
     #[derive(Clone, Copy, Debug, PartialEq, Eq)]
+    #[cfg_attr(feature = "defmt", derive(defmt::Format))]
```

Comment:
Should this be declared in `Cargo.toml`?

Reply:
The optional defmt feature should implicitely add a defmt feature (https://doc.rust-lang.org/cargo/reference/features.html)

Reply:
ok!